package.json

```diff
@@ -14,7 +14,7 @@
     "node": ">=6"
   },
   "dependencies": {
-    "prettier": "azz/prettier#f6d0b5fe"
+    "prettier": "prettier/prettier#ee148bfdeda373445ac9b67338cbae3df10a6576"
```

Comment:
maybe `prettier/prettier#master` since this is under dev anyway?

Reply:
Better to be explicit in package.json since it's locked in `yarn.lock` anyway.